lib/readline.js

```diff
@@ -1106,6 +1104,10 @@ Interface.prototype[SymbolAsyncIterator] = function() {
     const closeListener = () => {
       readable.push(null);
     };
+    const errorListener = (err) => {
+      readable.emit('error', err);
```

Comment:
`readable.destroy(err)`?

Reply:
Don't know how I missed this!

I've made the change.